src/DynamoCore/Search/NodeSearchModel.cs

```diff
@@ -172,17 +180,17 @@ private static void AddEntryToXml(XmlNode parent, NodeSearchElement entry, strin
             XmlHelper.AddNode(element, "LargeIcon", File.Exists(Path.Combine(dynamoPath, @"..\..\..\", pathToLargeIcon)) ? pathToLargeIcon : "Not found");
         }
 
-        private static void AddCategoryToXml(
+        private void AddCategoryToXml(
             XmlNode parent, ISearchCategory<NodeSearchElement> category, string dynamoPath)
         {
-            var element = XmlHelper.AddNode(parent, "Category");
-            XmlHelper.AddAttribute(element, "Name", category.Name);
+            //var element = XmlHelper.AddNode(parent, "Category");
+            //XmlHelper.AddAttribute(element, "Name", category.Name);
```

Comment:
Seems we can revert this change if we decide to not index based on xml node dump